### PR TITLE
Add "chore" type

### PR DESCRIPTION
**Pull request by AlexHodson**
Adds "chore" type to the default commit type options. Addresses #17

### CODE REVIEW

**Comment by sonyarianto** [comment]
@AlexHodson exactly what I want haha :) thanks.

File: .better-commits.json [hunk 1]
```diff
@@ -1,19 +1,80 @@
 {
```
Comment (Everduin94) [inline .better-commits.json:1]: Thanks @AlexHodson I appreciate the help. :)

Everything looks good. But you don't need to change this file. This is the config for *this* repository.

Anything that is not supplied gets replaced by the defaults at run-time. So you should see your change regardless of adding it here.

Comment (AlexHodson) [reply .better-commits.json]: Ahh, I see. I must have run the "init" npm script which caused the changes. I have reverted the better-commits.json file back to its original state.

Comment (Everduin94) [reply .better-commits.json]: Yup no worries, great job and thanks again! Merged.

**Pull request merged**